File: bank_app/app/src/main/java/com/test/banktest/worker/serviceWorker/BankService.java
package com.test.banktest.worker.serviceWorker;

import com.test.banktest.homeScreen.HomeResponse;
import com.test.banktest.loginScreen.LoginResponse;

import retrofit2.Call;
import retrofit2.http.Field;
import retrofit2.http.FormUrlEncoded;
import retrofit2.http.GET;
import retrofit2.http.Headers;
import retrofit2.http.POST;
import retrofit2.http.Path;

public interface BankService {

    @FormUrlEncoded
    @POST("login")
    @Headers({"Content-Type: application/x-www-form-urlencoded"})
    Call<LoginResponse> login(@Field("user") String user, @Field("password") String password);

    @GET("statements/{idUser}")
    @Headers({"Content-Type: application/x-www-form-urlencoded"})
    Call<HomeResponse> getStatements(@Path("idUser") int idUser);
}

File: app/src/main/kotlin/com/felipemsa/idletime/data/BankService.kt
package com.felipemsa.idletime.data

import retrofit2.Call
import retrofit2.http.*

interface BankService {

    @FormUrlEncoded
    @POST("login")
    fun login(@Field("user") user: String, @Field("password") pass: String): Call<LoginResponse>

    @GET("statements/{user_id}")
    fun statements(@Query("user_id") user: Int): Call<StatementsResponse>
}